elm.json

```diff
@@ -9,12 +9,21 @@
             "elm/browser": "1.0.2",
             "elm/core": "1.0.5",
             "elm/html": "1.0.0",
-            "elm/json": "1.1.3"
+            "elm/json": "1.1.3",
+            "elm/regex": "1.0.0",
+            "elm/svg": "1.0.1",
+            "elm-community/json-extra": "4.3.0",
+            "elm-community/list-extra": "8.7.0",
+            "elm-community/maybe-extra": "5.3.0",
```

Comment:
Have you considered using [`core-extra`](https://package.elm-lang.org/packages/elmcraft/core-extra/2.1.0/)? 😉